docs/examples/promise/06_make_transfer/index.js

```diff
@@ -16,9 +16,6 @@ async function main () {
   // Instantiate the API
   const api = await ApiPromise.create();
 
-  // Retrieve the nonce for Alice, to be used to sign the transaction
-  const aliceNonce = await api.query.system.accountNonce(alice.address());
-
   // Create a extrinsic, transferring 12345 units to Bob. We can also create,
   // sign and send in one operation (as per the samples in the Api documentation),
```

Comment:
This is probably just a leftover, possibly even from my big [Docs] PR. Could you remove the "... create sign and send in one operation..." sentence? :) Thanks!